index.d.ts

```diff
@@ -155,6 +155,4 @@ export interface Strava {
     oauth: OAuthRoutes;
 }
 
-declare const strava : Strava;
```

Comment:
We still need this line (but without the extra space after `strava`, if you don't mind

Reply:
@nik0kin Are you saying you cannot export it properly without it? Or what do you mean when saying we need this line? 

This line says says that the js module has a NAMED export `strava`, which it does not. So the typing pretends there is a variable that if used is actually undefined during runtime. This is the issue we ran into when using this lib the first time.

If we keep the line in question:
`import { strava } from 'strava-v3`

But `strava` is actually defined afterwards. Are you saying on your end its actually defined when importing via the above way?

Reply:
Thank you for pointing this out. Fixed.